qa/os/src/test/resources/http.crt

```diff
@@ -0,0 +1,62 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Does this cert have no expiration? How was it generated? Having hardcoded tests certs isn't necessarily bad, but if anyone ever needs to regenerate it, we need some kind of instructions on how it was made.

Reply:
Fair enough, I'll add a README in the resources dir